package/PartSeg/segmentation_analysis/main_window.py

```diff
@@ -561,6 +561,7 @@ def __init__(
         super().__init__(config_folder, title, settings, signal_fun)
         self.files_num = 2
         self.setMinimumWidth(600)
+        assert isinstance(self.settings, PartSettings)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/Czaki/PartSeg/pullRequest?prid=5714517)